src/index.ts

```diff
@@ -15,3 +19,23 @@ export function convertAxeToSarif(axeResults: Axe.AxeResults): SarifLog {
     const sarifConverter = defaultSarifConverter();
     return sarifConverter.convert(decoratedAxeResults, {});
 }
+
+export function sarifReporter(
+    rawResults: AxeRawResult[],
+    {},
+    callback: Function,
+) {
+    const environmentData: EnvironmentData = getEnvironmentDataFromEnvironment();
+    callback(createSarifReport(rawResults, environmentData));
+}
+
+function createSarifReport(
+    rawResults: AxeRawResult[],
+    environmentData: EnvironmentData,
+) {
+    return defaultAxeRawSarifConverter().convert(
+        rawResults,
+        {},
```

Comment:
Similarly here, I'd prefer to not pass in an empty object inline without any context to why it's here.

Either we can rename the function to be more appropriate (createSarifReport**WithoutOptions**?) or have an inline variable name for the options, to make it a bit easier for the reader.